commit 718e74e9b39bca38cf7cbd896c9f427d739c4c60
Author: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
Date:   Thu Nov 3 10:29:28 2016 -0600

    Fix: Disable sys_membarrier when nohz_full is enabled
    
    commit 907565337ebf998a68cb5c5b2174ce5e5da065eb upstream.
    
    Userspace applications should be allowed to expect the membarrier system
    call with MEMBARRIER_CMD_SHARED command to issue memory barriers on
    nohz_full CPUs, but synchronize_sched() does not take those into
    account.
    
    Given that we do not want unrelated processes to be able to affect
    real-time sensitive nohz_full CPUs, simply return ENOSYS when membarrier
    is invoked on a kernel with enabled nohz_full CPUs.
    
    Signed-off-by: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
    CC: Josh Triplett <josh@joshtriplett.org>
    CC: Steven Rostedt <rostedt@goodmis.org>
    Signed-off-by: Paul E. McKenney <paulmck@linux.vnet.ibm.com>
    Cc: Frederic Weisbecker <fweisbec@gmail.com>
    Cc: Chris Metcalf <cmetcalf@mellanox.com>
    Cc: Rik van Riel <riel@redhat.com>
    Acked-by: Lai Jiangshan <jiangshanlai@gmail.com>
    Reviewed-by: Josh Triplett <josh@joshtriplett.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1272_718e74e9b39bca38cf7cbd896c9f427d739c4c60.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1272_718e74e9b39bca38cf7cbd896c9f427d739c4c60.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
